### PR TITLE
DOCS-3617 Create Docs page for CLI on TrueNAS SCALE

### PR DESCRIPTION
**Pull request by tonyriv3**

### CODE REVIEW

File: content/SCALE/SCALETutorials/TrueNASCLIShell/_index.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+The CLI displays the message: "You have pending network interface changes. Please run 'network interface commit' to apply them."
+
+Enter `commit`, to apply the changes, then enter `checkin` to make them permanant. 
+
+![TrueNASCLIupdateinterfacealiases](/images/SCALE/TrueNASCLIupdateinterfacealiases.png "Update Interface Aliases")
+
+Enter `query` to make sure the Truenas applies the changes successfully.
+
+Enter `..` to exit interface and go up one level to the network menu.
```
Comment (clairewrtr) [inline content/SCALE/SCALETutorials/TrueNASCLIShell/_index.md:41]:
```suggestion
Enter `..` to exit the interface and go up one level to the network menu.
```

File: content/SCALE/SCALETutorials/TrueNASCLIShell/_index.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+#### Global Configuration
+
+This section covers configuring the defualt gateway.
```
Comment (clairewrtr) [inline content/SCALE/SCALETutorials/TrueNASCLIShell/_index.md:45]:
```suggestion
This section covers configuring the default gateway.
```

File: content/SCALE/SCALETutorials/TrueNASCLIShell/_index.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
+---
+title: "Using the TrueNAS CLI Shell"
+description: "This article describes how to use the SCALE CLI Shell for basic networking, updating, and storage management." 
+weight: 130
+---
```
Comment (clairewrtr) [inline content/SCALE/SCALETutorials/TrueNASCLIShell/_index.md:5]:
```tags:
- scaleinstall
---
```

File: content/SCALE/SCALETutorials/TrueNASCLIShell/_index.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+![TrueNASCLIstoragepoolquery](/images/SCALE/TrueNASCLIstoragepoolquery.png "Pool Query")
+
+Enter `q` to exit the query.
+
+To list all configured datasets, enter `storage dataset query`.
+
+![TrueNASCLIstoragedatasetquery](/images/SCALE/TrueNASCLIstoragedatasetquery.png "Dataset Query")
+
+Enter `q` to exit the query.
```
Comment (clairewrtr) [inline content/SCALE/SCALETutorials/TrueNASCLIShell/_index.md:77]:
```
Enter `q` to exit the query.

{{< taglist tag="scaleinstall" limit="10" >}}


```

**Review by clairewrtr — CHANGES_REQUESTED** [review]
Looks great, the comments re: tags and taglist are just suggestions.

File: content/SCALE/SCALETutorials/TrueNASCLIShell/_index.md (hunk 1 above)
Comment (tonyriv3) [reply content/SCALE/SCALETutorials/TrueNASCLIShell/_index.md:41]:
```suggestion
Enter `..` to exit `interface` and go up one level to the `network` menu.
```
interface is actually a menu in the CLI. I changed it to be more accurate.

**Review by clairewrtr — APPROVED** [review]

**Pull request merged**